lib/bus.js

```diff
@@ -1,12 +1,12 @@
 const isPlainObject = require('lodash/isPlainObject');
-const nanobus = require('nanobus');
+const EventEmitter = require('events');
```

Comment:
please revert this back to `nanobus`

Reply:
It would be good to include comments as to why `nanobus` is being used, when even their documentation would indicate that this project should use the native `EventEmitter`.

Regardless, `nanobus` luckily does have a `listeners()` API, so I'm able to work around the issue.